### PR TITLE
69.create teacher cabinet

### PR DESCRIPTION
**Pull request by Redingston**

### CODE REVIEW

File: src/app/pages/teacher-cabinet/teacher-cabinet.component.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    surname:"",
+    middleName: "",
+    phoneNumber: "",
+    email: "",
+    dateOfBirth: "",
+    imageUrl: "",
+    className: "",
+    degree: "",
+    schoolName: ""
+  };
```
Comment (DmMov) [inline src/app/pages/teacher-cabinet/teacher-cabinet.component.ts]: What need is it for?

File: src/app/components/teacher-info/teacher-info.component.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    surname:"",
+    middleName: "",
+    phoneNumber: "",
+    email: "",
+    dateOfBirth: "",
+    imageUrl: "",
+    className: "",
+    degree: "",
+    schoolName: ""
+  };
```
Comment (DmMov) [inline src/app/components/teacher-info/teacher-info.component.ts]: What need is it for?

File: src/app/components/teacher-info/teacher-info.component.scss [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        padding: 0.5rem;
+        width: 100%;
+
+        &:hover {
+          border-color: $accent;
+          background-color: #ffffff;
+          color: $accent;
+        }
+    }
+}
```
Comment (DmMov) [inline src/app/components/teacher-info/teacher-info.component.scss]: It is horrible

File: src/app/components/teacher-info/teacher-info.component.html [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    <button type="button" class="btn uploadPhoto primary">Upload</button>
+</div>
+<div class="name_edit">
+    <h2 class="full-name">{{user.surname}} {{user.firstName}} {{user.middleName}}</h2>
+    <h3 class="email">{{user.email}} </h3>
+    <h3 class="digit number">{{user.phoneNumber}} </h3>
+    <h3 class="date">{{user.dateOfBirth | slice:0:10 }} </h3>
+    <h3 class="class_teacher">Your class: {{user.className}} </h3>
+    <h3>{{user.degree}}</h3>
+</div>
```
Comment (DmMov) [inline src/app/components/teacher-info/teacher-info.component.html]: It is horrible as well, I mean the classes naming method

File: src/app/components/login-form/login-form.component.ts [hunk 5]
```diff
@@ -71,8 +71,6 @@ export class LoginFormComponent implements OnInit {
   onSubmit() {
     const { fields, isValid } = validationHelper(this.form.controls, this.fields);
 
-    this.fields = fields;
-
```
Comment (DmMov) [inline src/app/components/login-form/login-form.component.ts]: It is vital, why did you do that?
It was a huge mistake.

**Review by DmMov — COMMENTED** [review]
Sorry, but your request is trash

**Review by Karonuk — DISMISSED** [review]

**Review by shagi9 — DISMISSED** [review]

**Review by DmMov — APPROVED** [review]

**Review by Karonuk — APPROVED** [review]

**Review by shagi9 — APPROVED** [review]

**Pull request merged**